lib/cosmos/system/target.rb

```diff
@@ -167,9 +179,9 @@ def get_target_dir(path, name, gem_path)
         dir = File.join(path, name)
       end
       lib_dir = File.join(dir, 'lib')
-      Cosmos.add_to_search_path(lib_dir, false) if File.exist?(lib_dir)
+      Cosmos.add_to_search_path(lib_dir) if File.exist?(lib_dir)
```

Comment:
Sounds good. I think this makes a lot more sense now anyway since the target is namespaced you need to explicitly override it so system.txt should win.